packages/frontend/src/features/comments/components/CommentForm.tsx

```diff
@@ -0,0 +1,139 @@
+import { Comment } from '@lightdash/common';
+import {
+    Autocomplete,
+    AutocompleteItem,
+    Avatar,
+    Button,
+    Grid,
+    Group,
+    Stack,
+    Textarea,
+} from '@mantine/core';
+import { useForm } from '@mantine/form';
+import { IconAt } from '@tabler/icons-react';
+import { FC, useMemo, useState } from 'react';
+import MantineIcon from '../../../components/common/MantineIcon';
+import { useOrganizationUsers } from '../../../hooks/useOrganizationUsers';
+import { getNameInitials } from '../utils';
+
+type Props = {
+    userName: string;
+    onSubmit: (text: string) => Promise<null>;
+    isSubmitting: boolean;
+    onCancel?: () => void;
+    mode?: 'reply' | 'new';
+};
+
+export const CommentForm: FC<Props> = ({
+    userName,
+    onSubmit,
+    isSubmitting,
+    onCancel,
+    mode = 'new',
+}) => {
+    const commentForm = useForm<Pick<Comment, 'text' | 'replyTo'>>({
+        initialValues: {
+            text: '',
+            replyTo: '',
+        },
+        validate: {
+            text: (value) => {
+                if (value.trim() === '') {
+                    return 'Comment cannot be empty';
+                }
+                return null;
+            },
+        },
+    });
+    const { data: listUsers } = useOrganizationUsers();
```

Comment:
we might want to get the project users, no? 🤔

Reply:
mmm, I didn't think about that. This is what we already use on user invite (I don't think we have an enpdoint for project users) 
Project users is a subset of organization users. in fact. Project users = org users - members (without project role)

So if you send a message to a user that doesn't have access to a project:
- this user will see a notification about a mention
- if he  clicks on the notification he will get an error saying he doesn't have access

The question is, is that a good thing ? or not. The user can ask the admin for permission if he finds out he doesn'tn have access

On the other hand, if we only show project users, and I want to send a message to a person I know is on the org, but doesn't have access to a project, I'll be wondering why he doesn't appear on the list...